README.md

```diff
@@ -13,10 +13,9 @@ Follow [these instructions](https://docs.docker.com/install/) to get Docker runn
 
 ## Quickstart
 
-To get started using pre-built [Docker Hub grocy images](https://hub.docker.com/u/grocy), run the following commands:
+To get started using pre-built [Docker Hub grocy images](https://hub.docker.com/u/grocy), run the following command:
 
 ```sh
-docker-compose pull
```

Comment:
What does `docker-compose pull` do if the `docker-compose.yml` file has been modified locally?

Reply:
This is an error in the commit. I was actually trying to delete this command and keep only docker-compose up in the read me precisely because it doesn't actually do anything.